### DISCUSSION TITLE
[Idea] Push Notification Capabilities

### DISCUSSION THREAD

**Discussion by tomtom215** [mmcc-xx/WhosAtMyFeeder#6] [Ideas]
First off, a huge thank you for your efforts and making this project open source! This is something I have wanted for a long time and had the hardware for, but never got something working as smoothly as you have.

I was wondering if you had an opinion on setting up a notification framework? 

My idea would be you could configure different notifications, some examples could be:

- new detection (maybe with ability to batch in groups of 5-10)
- new (not seen yet) species detection
- daily summary report sent at end of day 
- monthly report

I think this would make it even more "set and forget" because then you can just get notified of the birds via a push notification to your phone instead of having to go to the web-app every time to check stats.

A great project to achieve this would be : https://github.com/caronc/apprise

**Comment by mmcc-xx** [comment]
I'm glad you are enjoying it! I just added notification support via apprise to the other app I'm working on which detects bird via sound. At some point, I really need to marry these two apps together because, especially if someone is using a security camera for detecting via video, it is likely to include audio as well.

Anyhow, that said my plan is to include some basic support for notifications, but also support a decent MQTT publication scheme so that if someone wanted to fine tune a notification scheme they could do so using data from MQTT.
